python/tvm/topi/x86/nn.py

```diff
@@ -69,3 +69,33 @@ def schedule_softmax(outs):
         s[exp].compute_at(s[softmax], fused_outer_axes)
 
     return s
+
+
+def schedule_embed_grad(outs):
+    """Schedule for embed_grad
+
+    Parameters
+    ----------
+    outs: Array of Tensor
+          The computation graph description of embed_grad
+          in the format of an array of tensors.
+
+    Returns
+    -------
+    sch: Schedule
+        The computation schedule for the op.
+    """
+    s = te.create_schedule([outs[0].op])
+
+    vec_size = 8  # should autotune this, but we can't with hybrid script
```

Comment:
May I ask why 8? I am just wondering if we could reuse this schedule for the arm back-end as well

Reply:
We could reuse it. I just didn't have a good way to figure out the width of the vector instructions.